connectors/src/connectors/slack/chat/blocks.ts

```diff
@@ -98,25 +102,117 @@ function makeContextSectionBlocks(
   return resultBlocks;
 }
 
+function makeAssistantSelectionBlock(
+  agentConfigurations?: LightAgentConfigurationType[]
+) {
+  return agentConfigurations
+    ? [
+        {
+          type: "actions",
+          block_id: "agentConfigId",
+          elements: [
+            {
+              type: "static_select",
+              placeholder: {
+                type: "plain_text",
+                text: "Switch to another assistant",
+                emoji: true,
+              },
+              options: agentConfigurations.map((ac) => {
```

Comment:
`nit` can we only take favorites? To avoid sending all of them?

Reply:
you mean most popular ? the ones with most usage ?

Reply:
Yep!